MovieTime/MovieTimeTests/MovieTimeTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MovieTimeTests.swift
+//  MovieTimeTests
+//
+//  Created by Enrique Melgarejo on 05/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import XCTest
+@testable import MovieTime
+
+class MovieTimeTests: XCTestCase {
```

Comment:
if you had more time, what tests would you add to this class?

Reply:
I would test all *ViewModels*, because that's where the business rules are, the `MTMovieDBImageService` image service and the application's core classes:
- Coordinators;
- MTNetworkError
- MTServiceHelper
- MTParser<T>